.github/workflows/bump-releases.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build
+
+on:
+  schedule:
+  - cron: 0 0 * * *    # every day at midnight
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: hmarr/debug-action@v1.0.0
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Configure git configs
```

Comment:
Is this needed? Git should already be configured properly inside the worker.

Reply:
I'd rather do it here such that we can reuse the bump_external_releases tool also on our laptops in case something does not work as expected